.github/workflows/codeql-analysis.yml

```diff
@@ -37,12 +56,6 @@ jobs:
         # https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#overriding-automatic-language-detection
 
     steps:
-      - id: harden-runner
```

Comment:
I do not believe this should be removed.

Reply:
Doh!  You are correct.  Fixed in commit 2e30384.